frontend/components/puzzleA.jsx

```diff
@@ -83,8 +83,8 @@ class PuzzleA extends React.Component {
     const solved = isSolved(gameState, 'A');
 
     const question1Icons = ['chevron-circle-up', 'chevron-circle-down', 'chevron-circle-left', 'chevron-circle-down'];
```

Comment:
looks like we have images that chris provided here, can you add those too?

Reply:
What do you think of the images I have added? Look okay?